modules/odc-viewing-application-topology.adoc

```diff
@@ -5,29 +5,24 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
-
-
-The status or phase of the Pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
-
-After you create an application and an image is deployed, the status is shown as *Pending*. After the application is built, it is displayed as *Running*.
-
-.Application topology
-image::odc_application_topology.png[]
-
-The application resource name is appended with indicators for the different types of resource objects as follows:
-
-* *CJ*: `CronJob`
-* *D*: `Deployment`
-* *DC*: `DeploymentConfig`
-* *DS*: `DaemonSet`
-* *J*: `Job`
-* *P*: `Pod`
-* *SS*: `StatefulSet`
-* image:odc_serverless_app.png[title="Serverless Application"] (Knative): A serverless application
-+
-[NOTE]
-====
-Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Topology* view. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Topology* view.
-====
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+
+The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the graph view.
```

Comment:
s/monitor your applications using the *List* view./monitor your applications using the list view.
*List View*/*List view* view is in small case on the UI. Keep the names of GUI elements identical to the way it is displayed on the GUI.
*Topology View*/ *Topology view*

Reply:
Will incorporate the changes suggested in the new commit.